web-ui/src/main/java/com/tutrit/webclient/controller/CustomerController.java

```diff
@@ -0,0 +1,53 @@
+package com.tutrit.webclient.controller;
+
+import com.tutrit.bean.Customer;
+import com.tutrit.gateway.CustomerGateway;
+import jakarta.servlet.http.HttpServletRequest;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.servlet.ModelAndView;
+
+import java.util.Optional;
+
+@Controller
+public class CustomerController {
+    @Autowired(required = false)
+    private CustomerGateway customerGateway;
+
+    @GetMapping("customers/{customerId}")
+    public ModelAndView findCustomerById(@PathVariable String customerId) {
+        var mov = new ModelAndView();
+        customerGateway.findCustomerById(customerId)
+                .ifPresentOrElse(
+                        c -> mov.addObject("customer", c),
+                        () -> {
+                            mov.addObject("customer", new Customer(null, null, null, null, null));
+                            mov.addObject("error", "Customer not found");
+                        });
+
+        mov.setViewName("customer-form.html");
+        return mov;
+    }
+
+    @PostMapping("/customers/{id}")
```

Comment:
I think we also need 
```java
@PostMapping("/customers")
```
endpoint that will handle creation of a new customer.

Reply:
I create new task for the solving this issue
https://schneidervs.atlassian.net/browse/TGBOT-329